frontend/src/pages/Common/CreationDialog.js

```diff
@@ -7,6 +7,14 @@ import DialogTitle from "@material-ui/core/DialogTitle";
 import isEmpty from "lodash/isEmpty";
 import strings from "../../localizeStrings";
 import CreationDialogStepper from "./CreationDialogStepper";
+import { withStyles } from "@material-ui/core";
```

Comment:
Dont destructure in import since it is importing the whole "core" lib

Reply:
```suggestion
import { withStyles } from "@material-ui/core/styles";
```